test/jdk/java/awt/Graphics/TextAAHintsTest.java

```diff
@@ -139,11 +163,85 @@ public Dimension getPreferredSize() {
         return new Dimension(500,300);
     }
 
-    public static void main(String[] args) throws Exception {
+    public static void createTestUI() {
+        frame = new Frame("Composite and Text Test");
+        TextAAHintsTest textAAHintsTestObject = new TextAAHintsTest();
+        frame.add(textAAHintsTestObject, BorderLayout.NORTH);
+
+        String instructions = """
+                Note: Texts are rendered with different TEXT_ANTIALIASING &
+                   VALUE_TEXT_ANTIALIAS. Text should be B&W, grayscale, and LCD.
+                   Note: The results may be visually the same.
+                1. Verify that first set of text are rendered correctly.
+                2. Second set of text are created using BufferedImage of the first text.
+                3. Third set of text are created using VolatileImage of the first text.
+                """;
+        TextArea instructionTextArea = new TextArea(instructions, 8, 50);
+        instructionTextArea.setEnabled(false);
```

Comment:
Please leave it enabled but make it read-only (`setEditable(false)`). This way the text will be black and easier to read.

Reply:
Done